### DISCUSSION TITLE
Hawaii Hydrojet Pro S200102 6 wires not working properly

### DISCUSSION THREAD

**Discussion by esys4u** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#648] [General]
![](https://user-images.githubusercontent.com/97408385/277624359-59041390-54ba-4f5d-812c-70e35123bdd0.png)
Hi, I had Hawaii Hydrojet Pro S200102 6 wires and setup PCB_V2B + red LLC, no resistors, software from release v4.2.3, and config as below on screens:

The temperature is not correct, touch panel is not working, running buubles is working but with big delay ( the button is not stay in position ON after push but pump is starting after few seconds od delay ) any ideas what need to be checked? 

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/97408385/080733e2-ab8e-4cea-872f-a9c283845922)

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/97408385/78bda5c0-1427-42d3-97de-c2d93dd81bae)

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/97408385/8ce9f3a7-c146-4af5-bfc6-a3269a0e1b57)

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/97408385/4abfe319-cef6-42b3-8b9c-2925883bca0c)

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/97408385/92a1b73d-ba25-4e21-894b-0ea65c430770)

also hardware test is OK
![Zrzut ekranu 2023-10-24 o 12 07 44](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/97408385/64b8a3ff-23b6-452c-9b91-93c96864eaee)

**Comment by visualapproach** [comment]
Is it a model that already has Wi-Fi? In that case I'm afraid it won't work. At least that's what I have heard from some people

**Reply by esys4u** [reply]
no it's without wifi

**Reply by visualapproach** [reply]
Ok maybe you need the low pass filters. Thanks for providing all info by the way.

**Reply by esys4u** [reply]
@visualapproach I should install capacitors and resistors or just only capacitors in this setup ?

**Reply by visualapproach** [reply]
LP filter is cap+resistor.

**Reply by esys4u** [reply]
@visualapproach On instruction I see 10K ohms but on the V2B board is writed that I should use ~560 Ohms and 200pF Cap what is correct for this setup?

**Reply by visualapproach** [reply]
Both. As printed on the PCB you use the R1/10K if you have "the blue" level converter. Nothing to do with the filters that should have around 560 Ohms.

**Reply by esys4u** [reply]
still the same behavior, pump panel is not working :( any ideas how to debug this ? local wire test is also OK for all pins.

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/97408385/9bb605d9-1ade-463c-af06-be3dd99f012b)

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/97408385/203c0da5-a664-4136-8f1b-94b82e1e5683)

**Reply by esys4u** [reply]
also app UI is not working correctly, buttons are not working the same as in this picture from project 
![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/97408385/bf59bd91-c47a-4172-af29-b343e95cb02d)

**Reply by visualapproach** [reply]
How is the ui not working? 404 error or just that the buttons don't do anything?
